### PR TITLE
refactor(build): make the org configurables

### PR DESCRIPTION
**Pull request by kmova**
Removing a few more hardcoded references to openebs, seeing during pushing jiva docker image. 

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

**Comment by kmova** [comment]
Tested via: https://github.com/mayadata-io/jiva/pull/2

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,11 @@ IS_DOCKER_INSTALLED       := $(shell which docker >> /dev/null 2>&1; echo $$?)
 
 TARGETS := $(shell ls scripts)
 
+ifeq (${IMAGE_ORG}, )
+  IMAGE_ORG = openebs
```
Comment (vishnuitta) [inline Makefile:17]: is this supposed to be ":="? seeing this at all the other places

Comment (kmova) [reply Makefile:17]: Checked this on. #TIL. 
When making direct string assignments, `:=` is not required. not required.  `:=`  is used in other places to expand expression during definition.

Comment (vishnuitta) [reply Makefile:17]: both the forms does the expansion.. '=' makes recursive ones which can lead to infinite loop, where as, ':=' does expansion only once..
for direct assignments, both should be fine (https://www.gnu.org/software/make/manual/make.html#Flavors)

**Pull request merged**